client/src/app/components/forms/admin-table/admin-table.component.ts

```diff
@@ -38,4 +43,14 @@ export class AdminTableComponent implements OnInit {
 	onSave(element: RecursivePartial<guestData>) {
 		this.updateGuest.emit(element);
 	}
+
+	onExpand() {
+		if (this.expanded) {
```

Comment:
personally, i'd do this function like

this.expanded = !this.expanded
and set displayedColumns in a ternary

because i find it a bit more readable? up to you tho lol, its very...nitpicky

Reply:
Ah yeah lol